untitled/src/homework2/task1.java

```diff
@@ -0,0 +1,29 @@
+package homework2;
+
+import java.util.Scanner;
+
+//Tasks
+public class task1 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please add char ");
+        char c = sc.next().charAt(0);
+
+    switch (c) {
```

Comment:
It is not wrong, but you could reuse the code:
        case 'Y' :
        case 'y' :
            System.out.println("Yes");
            break;
        case 'N' :
        case 'n' :
            System.out.println("No");
            break;